doc/Doxyfile.in

```diff
@@ -247,7 +247,18 @@ ALIASES                = "signal=- @ref  " \
                          "endsignalproto=@endcode  " \
                          "signaldesc=" \
                          "signals=@b Signals:  " \
-                         "endsignals=  "
+                         "endsignals=  " \
+                         "addtogir=@internal"
```

Comment:
should probably rather be named something like `gitonly` as it's not very clear now when it appears in the docs